components/suggestion/search_date_suggestion/search_date_suggestion.tsx

```diff
@@ -16,18 +16,19 @@ import 'moment';
 const loadedLocales: Record<string, moment.Locale> = {};
 
 type Props = {
-    locale: string;
+    locale?: string;
+    currentDate?: any;
 }
 
 export default class SearchDateSuggestion extends Suggestion {
     handleDayClick = (day: Date) => {
         const dayString = day.toISOString().split('T')[0];
-        this.props.onClick(dayString, this.props.matchedPretext);
+        this.props.onClick!(dayString, this.props.matchedPretext);
```

Comment:
It more readable to just check for undefined
```suggestion
        if (this.props.onClick) {
            this.props.onClick(dayString, this.props.matchedPretext);
        }
```

Reply:
Actually we can do this even better with a coalesce operator
```suggestion
        this.props.onClick?.(dayString, this.props.matchedPretext);
```